update korean translation

leesongun opened the pull request.

hauensteina commented:
Can you explain what improvements you made to the translations?
Were there mistakes?

leesongun commented:
translation of `About`, `Best`, `Score`, `Fast`, `Strong`, `Settings`, `passes`, `Click` was phonetic transcription(it's like using words like `sushi` or `Go`  even though you already have words for it)


because korean can rarely omit units, i updated `Fwd 10` and `Back 10` to `앞으로 10수(forward 10 moves)` and `뒤로 10수(backward 10 moves)` respectively

The only mistranslation was `Medium`, as `매질` means `medium` as in `medium of wave` 

in translation of  `Show probability during play`, `가능성 퍼센티지` means `probability percentage`, which seemed like duplicate

`빠른`, `느린` is an adjective(관형사) form, so i changed it with noun(명사) form, `빠름`, `느림`
